### ISSUE TITLE
Network has been temporary banned - BAD REQUEST

### ISSUE THREAD

**Issue by Vinibch** [jsanahuja/jquery.instagramFeed#99]
Hello !
Is a problem :

```
InstagramFeed: It looks like YOUR network has been temporary banned because of too many requests. 
See https://github.com/jsanahuja/jquery.instagramFeed/issues/25 4
```

and 

```
Uncaught TypeError: Cannot set property 'innerHTML' of null
    at Object.display (InstagramFeed.min.js:1)
    at InstagramFeed.min.js:1
    at XMLHttpRequest.c.onload (InstagramFeed.min.js:1)
```

**Comment by mahotilo** [comment]
It means exactly what it says. You are temporarily banned by IG. Wait at least for 24 hours since the last script activation and try again.

**Comment by Vinibch** [comment]
J'ai trouvé une meilleure alternative :
https://github.com/stevenschobert/instafeed.js/

**Comment by mahotilo** [comment]
It is not an alternative because of 
` Instagram Basic Display API`

**Comment by Vinibch** [comment]
Exactly, but the solution is less limited for my ecommerce use ;)

**Comment by mahotilo** [comment]
But you should take into account that only you are banned, and your visitors can see IG images.

**Comment by jsanahuja** [comment]
Closing as the new client-side caching (`cache_time`) feature implemented in #97 (2.0.0 release) should deal with this.

**Comment by Vinibch** [comment]
Great Jog. I will test the update tomorrow
